src/components/AboutHome/AboutHome.js

```diff
@@ -49,6 +52,21 @@ const AboutHome = ({ user, setCurrentLink }) => {
     );
   }, []);
 
+  //Display login warnings for the login initial authorizer user with no responsibilities.
+  useEffect(() => {
+    //get the current user
+    const cdxUser = JSON.parse(localStorage.getItem("ecmps_user"));
+
+    //check if the user has initial authorizer role with no responsibilities
+    if (cdxUser?.roles?.includes(config.app.initialAuthorizerRole) &&  JSON.stringify(cdxUser?.facilities) === '{}')
```

Comment:
Doing 'JSON.stringify(cdxUser?.facilities)' to just check if facilities is empty may have overhead (among other things).  It also doesn't catch cases where facilities is null or undefined. 

Better alternative:  Object.keys(cdxUser?.facilities || {}).length === 0

For bypass users with no facilities, I think cdxUser.facilities will be null.  Can you check if you are supposed to display the warning message in such cases?

Reply:
Hi Yonatan, the alternative way you provide is a better way to handle facilities empty/null/undefined situation. I will update them based on your notes. 
Also, Mark confirmed that we should be consistent in the behavior, so we will need to display the warning message for bypass users with no facilities.